### DISCUSSION TITLE
yolov8노드에서 퍼블리쉬 과정 중에 에러가 떴습니다.

### DISCUSSION THREAD

**Discussion by po2955** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#41] [2024년도 교육 자료 관련 질문] [answered]
![스크린샷 2024-07-13 14-10-37](https://github.com/user-attachments/assets/a9f14447-7151-4be3-8266-779b32661f19)

다음과 같은 에러가 떠서 구글링 하면서 수정 해봤는데 고쳐지지가 않습니다. 도움 부탁드리겠습니다.

**Comment by Jinsun-Lee** [comment]
![image](https://github.com/user-attachments/assets/1e2f82c8-7f98-471d-a41c-d3dd92419cbf)
```
python -c "import torch; print(torch.__version__)"
```
위 명령어를 실행하고 결과를 공유해 주시면 더 적합한 도움을 드릴 수 있을 것 같습니다.

**Reply by po2955** [reply]
![스크린샷 2024-07-14 18-48-59](https://github.com/user-attachments/assets/1cc248a1-5b59-4729-b71f-c0a7c5d33f11)

이렇게 나왔습니다!

**Reply by HyeongKeunHong** [reply]
```
pip install torch==2.2.0+cu121
```
위 명령어를 실행해보고 다시
```
python -c "import torch; print(torch.__version__)"
```
를 실행해보시겠어요?

**Reply by po2955** [reply]
용량이 부족해서 설치가 안됩니다.
영상따라 30G로 파티션 할당 했는데 오류 수정하려 이것 저것 다운 받다가 용량이 꽉 찼나봐요.
포맷하고 처음부터 다시 하는게 나을까요?

**Reply by HyeongKeunHong** [reply, marked as answer]
예, 포맷하고 처음부터 다시하는게 오히려 빠를 것 같습니다. 하시다가 문제 생기면 다시 새로운 글로 문의주시면 되겠습니다.

**Reply by po2955** [reply]
네 알겠습니다!
